src/mavsdk/plugins/mission_raw_server/include/plugins/mission_raw_server/mission_raw_server.h

```diff
@@ -185,6 +195,17 @@ class MissionRawServer : public ServerPluginBase {
      */
     void unsubscribe_incoming_mission(IncomingMissionHandle handle);
 
+    /**
+     * @brief Subscribe to the result of outgoing mission download requests (air to ground)
+     */
+    OutgoingMissionResultHandle
+    subscribe_outgoing_mission_result(const OutgoingMissionResultCallback& callback);
```

Comment:
I added the ability to get notifications when a mission download request completes (with status) because it seemed harmless. In practice I'm not sure how useful this will actually be, but I left it here, happy to take it out if you think it's cluttering the API.